rest/common.ts

```diff
@@ -103,7 +103,7 @@ export enum RESTJSONErrorCodes {
 	GuildAlreadyHasTemplate = 30031,
 	MaximumNumberOfApplicationCommandsReached,
 	MaximumThreadParticipants,
-
+	MaximumDailyApplicationCommandCreates,
 	MaximumNumberOfNonGuildMemberBansHasBeenExceeded = 30035,
```

Comment:
This = is no longer needed then

Reply:
huh? why?

Reply:
For the same reason you don't need it for MaximumNumberOfApplicationCommandsReached, MaximumThreadParticipants and now MaximumDailyApplicationCommandCreates. It autoincrements

Reply:
Oh, right. I will close this, thanks.

Reply:
Thats not what I meant! This error needs to be documented, its just that MaximumNumberOfNonGuildMemberBansHasBeenExceeded doesn't need to be explicitly declared as 30035!

Reply:
Pretty sure he meant the value isn’t needed, no reason to close the PR 🤔

Reply:
Oh, so sorry. I misread all 🤦‍♂️. I should have removed that before, I will change that!

Reply:
Done!